classes/userGroup/relationships/UserGroupStage.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/**
+ * @file classes/userGroup/relationships/UserUserGroup.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class \PKP\userGroup\UserGroup
+ * @ingroup userGroup
+ *
+ * @see DAO
+ *
+ * @brief UserGroup metadata class.
+ */
+
+namespace PKP\userGroup\relationships;
+
+use PKP\facades\Locale;
+use PKP\db\DAORegistry;
+use PKP\user\User;
+use PKP\userGroup\UserGroup;
+use APP\facades\Repo;
+use Illuminate\Database\Eloquent\Casts\Attribute;
+use Illuminate\Database\Eloquent\Builder;
+
+class UserGroupStage extends \Illuminate\Database\Eloquent\Model
+{
+    //
+    // The following attributes configure Eloquent's expectations for this model
+    //
+    public $timestamps = false;
+    public $incrementing = false;
+    protected $primaryKey = null;
+    protected $fillable = ['userGroupId', 'stageId', 'contextId'];
+
+    /**
+     * The table associated with the model.
+     *
+     * @var string
+     */
+    protected $table = 'user_group_stage';
+
+    public function userGroup(): Attribute
+    {
+        return Attribute::make(
+            get: fn($value, $attributes) => Repo::userGroup()->get($attributes['user_group_id']),
+            set: fn($value) => $value->getId()
+        );
+    }
+
+    //
+    // The following two functions permit lowerCamelCase accessors in place of snake_case column names
```

Comment:
This is probably borrowed from my example Eloquent model; there are 3 of them in this case, but best to just remove these comments, I think.

Reply:
Removed from 
- `UserGroupStage`
- `UserUserGroup`